po/fr.po

```diff
@@ -0,0 +1,206 @@
+#: data/resources/ui/preferences.ui:73
+msgid "HSV"
+msgstr "HSV"
+
+#: data/com.github.finefindus.eyedropper.metainfo.xml.in.in:20
+msgid "CMYK"
+msgstr "CMYK"
+
+#: data/com.github.finefindus.eyedropper.metainfo.xml.in.in:21
+#: data/resources/ui/preferences.ui:97
+msgid "XYZ"
+msgstr "XYZ"
+
+#: data/com.github.finefindus.eyedropper.metainfo.xml.in.in:22
+#: data/resources/ui/preferences.ui:109
+msgid "CIELAB"
+msgstr "CIELAB"
+
+#: data/com.github.finefindus.eyedropper.metainfo.xml.in.in:28
+msgid "Main window dark"
+msgstr "Fenêtre principale sombre"
+
+#: data/com.github.finefindus.eyedropper.metainfo.xml.in.in:32
+msgid "Main window light"
+msgstr "Fenêtre principale claire"
+
+#: data/com.github.finefindus.eyedropper.metainfo.xml.in.in:36
+#: data/com.github.finefindus.eyedropper.metainfo.xml.in.in:40
+msgid "Customized color formatting"
+msgstr "Formatage personnalisé des couleurs"
+
+#: data/com.github.finefindus.eyedropper.metainfo.xml.in.in:76
+msgid "FineFindus"
+msgstr "FineFindus"
+
+#: data/resources/ui/preferences.ui:10
+msgid "General"
+msgstr "Général"
+
+#: data/resources/ui/preferences.ui:13
+msgid "Format"
+msgstr "Format"
+
+#: data/resources/ui/preferences.ui:16
+msgid "Alpha-Value-Position"
+msgstr "Position de la valeur alpha"
+
+#: data/resources/ui/preferences.ui:17
+msgid "Where the Alpahvalue is positioned in the Hexstring"
+msgstr "Position de la valeur alpha dans la chaîne hexadécimale"
+
+#: data/resources/ui/preferences.ui:21
+msgid "None"
+msgstr "Aucune"
+
+#: data/resources/ui/preferences.ui:22
+msgid "End"
+msgstr "Fin"
+
+#: data/resources/ui/preferences.ui:23
+msgid "Start"
+msgstr "Début"
+
+#: data/resources/ui/preferences.ui:33
+msgid "Color Formats"
+msgstr "Formats de couleur"
+
+#: data/resources/ui/preferences.ui:34
+msgid "Choose which formats will be visible"
+msgstr "Choisissez quels formats seront visibles"
+
+#: data/resources/ui/preferences.ui:37
+msgid "Hex Code"
+msgstr "Code hexadécimal"
+
+#: data/resources/ui/preferences.ui:85
+msgid "CMKY"
+msgstr "CMYK"
+
+#: data/resources/ui/shortcuts.ui:11
+msgctxt "shortcut window"
+msgid "General"
+msgstr "Général"
+
+#: data/resources/ui/shortcuts.ui:14
+msgctxt "shortcut window"
+msgid "Show Preferences"
+msgstr "Afficher les préférences"
+
+#: data/resources/ui/shortcuts.ui:20
+msgctxt "shortcut window"
+msgid "Show Shortcuts"
+msgstr "Afficher les raccourcis"
+
+#: data/resources/ui/shortcuts.ui:26
+msgctxt "shortcut window"
+msgid "Quit"
+msgstr "Quitter"
+
+#: data/resources/ui/shortcuts.ui:34
+msgctxt "shortcut window"
+msgid "Color"
+msgstr "Couleur"
+
+#: data/resources/ui/shortcuts.ui:37
+msgctxt "shortcut window"
+msgid "Pick a new Color"
+msgstr "Choisir une nouvelle couleur"
+
+#: data/resources/ui/shortcuts.ui:43
+msgctxt "shortcut window"
+msgid "Randomize the Color"
+msgstr "Randomiser la couleur"
+
+#: data/resources/ui/window.ui:6
+msgid "_Random Color"
+msgstr "Couleur _aléatoire"
```

Comment:
The underscores should be at the start, they are only used by GTK to indicate that the shortcut has a keyboard accelerator. Same with the 3 other shortcuts

Reply:
I understand your point, but the French keyboard shortcuts are highly codified by the GNOME translation team, to avoid duplication of letters, to create continuity between each application, and to avoid special characters (like "À") that are difficult to type on the keyboard.
As a result, as far as the position of underscores is concerned, I think it's better to leave them as they are in order to follow the translation team's guidelines as best as possible (besides, it's common to see underscores elsewhere than at the beginning of the string in French menu translations)...

Reply:
Interesting, I did not know that.